projects/plugins/jetpack/modules/masterbar/admin-menu/class-admin-menu.php

```diff
@@ -574,31 +354,20 @@ function ( $parent_file ) use ( $profile_slug ) {
 	 * @param bool $wp_admin_export Optional. Whether Export link should point to Calypso or wp-admin. Default false (Calypso).
 	 */
 	public function add_tools_menu( $wp_admin_import = false, $wp_admin_export = false ) {
-		$admin_slug = 'tools.php';
-		$menu_slug  = 'https://wordpress.com/marketing/tools/' . $this->domain;
-
-		remove_menu_page( $admin_slug );
-		remove_submenu_page( $admin_slug, $admin_slug );
-		remove_submenu_page( $admin_slug, 'delete-blog' );
-		remove_submenu_page( $admin_slug, 'import.php' );
-		remove_submenu_page( $admin_slug, 'export.php' );
-
-		add_menu_page( esc_attr__( 'Tools', 'jetpack' ), __( 'Tools', 'jetpack' ), 'publish_posts', $menu_slug, null, 'dashicons-admin-tools', 75 );
-		add_submenu_page( $menu_slug, esc_attr__( 'Marketing', 'jetpack' ), __( 'Marketing', 'jetpack' ), 'publish_posts', $menu_slug );
-		add_submenu_page( $menu_slug, esc_attr__( 'Earn', 'jetpack' ), __( 'Earn', 'jetpack' ), 'manage_options', 'https://wordpress.com/earn/' . $this->domain );
-		add_submenu_page( $menu_slug, esc_attr__( 'Import', 'jetpack' ), __( 'Import', 'jetpack' ), 'import', $wp_admin_import ? 'import.php' : 'https://wordpress.com/import/' . $this->domain );
-		add_submenu_page( $menu_slug, esc_attr__( 'Export', 'jetpack' ), __( 'Export', 'jetpack' ), 'export', $wp_admin_export ? 'export.php' : 'https://wordpress.com/export/' . $this->domain );
-
-		$this->migrate_submenus( $admin_slug, $menu_slug );
+		$submenus_to_update = array();
+		if ( ! $wp_admin_import ) {
+			$submenus_to_update['import.php'] = 'https://wordpress.com/import/' . $this->domain;
+		}
+		if ( ! $wp_admin_export ) {
+			$submenus_to_update['export.php'] = 'https://wordpress.com/export/' . $this->domain;
+		}
+		$this->update_submenus( 'tools.php', $submenus_to_update );
 
-		add_submenu_page( $menu_slug, esc_attr__( 'Other tools', 'jetpack' ), __( 'Other tools', 'jetpack' ), 'manage_options', 'tools.php' );
```

Comment:
We need to add "Other Tools" too

![](https://cln.sh/bb2sYx+)

Reply:
This removal is intentional, see https://github.com/Automattic/wp-calypso/issues/50567#issuecomment-798770921. This helps partially fixing https://github.com/Automattic/wp-calypso/issues/50193.